docs/examples/linkedin.rst

```diff
@@ -9,31 +9,39 @@ command line interactive example below.
 
 .. code-block:: pycon
 
+    >>> # Imports
+    >>> import os
+    >>> from requests_oauthlib import OAuth2Session
+
+    >>> # Set environment variables
+    >>> os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'
+
     >>> # Credentials you get from registering a new application
     >>> client_id = '<the id you get from linkedin>'
     >>> client_secret = '<the secret you get from linkedin>'
 
-    >>> # OAuth endpoints given in the LinkedIn API documentation
-    >>> authorization_base_url = 'https://www.linkedin.com/uas/oauth2/authorization'
-    >>> token_url = 'https://www.linkedin.com/uas/oauth2/accessToken'
+    >>> # LinkedIn OAuth2 requests require scope and redirect_url parameters.
+    >>> # Ensure these values match the auth values in your LinkedIn App 
+    >>> # (see auth tab on LinkedIn Developer page)
+    >>> scope = ['r_liteprofile']
+    >>> redirect_url = 'http://127.0.0.1'
 
-    >>> from requests_oauthlib import OAuth2Session
-    >>> from requests_oauthlib.compliance_fixes import linkedin_compliance_fix
+    >>> # OAuth endpoints given in the LinkedIn API documentation
+    >>> authorization_base_url = 'https://www.linkedin.com/oauth/v2/authorization'
+    >>> token_url = 'https://www.linkedin.com/oauth/v2/accessToken'
 
-    >>> linkedin = OAuth2Session(client_id, redirect_uri='http://127.0.0.1')
-    >>> linkedin = linkedin_compliance_fix(linkedin)
+    >>> linkedin = OAuth2Session(client_id, redirect_uri='http://127.0.0.1', scope=scope)
 
     >>> # Redirect user to LinkedIn for authorization
     >>> authorization_url, state = linkedin.authorization_url(authorization_base_url)
-    >>> print 'Please go here and authorize,', authorization_url
+    >>> print("Please go here and authorize, {}".format(authorization_url))
```

Comment:
Have you thought about using [f-strings, introduced in Python 3.6](https://docs.python.org/3/whatsnew/3.6.html#pep-498-formatted-string-literals)? They're really handy, and might make this easier to read:

```suggestion
    >>> print(f"Please go here and authorize: {authorization_url}")